packages/core/src/layout/InfoCard/InfoCard.stories.tsx

```diff
@@ -15,29 +15,42 @@
  */
 import React from 'react';
 import InfoCard from '.';
+import { Grid } from '@material-ui/core';
 
-const cardContentStyle = { height: '200px' };
+const cardContentStyle = { height: 200, width: 500 };
 const linkInfo = { title: 'Go to XYZ Location', link: '#' };
 
 export default {
   title: 'Information Card',
   component: InfoCard,
 };
 
+const Wrapper = ({ children }) => (
+  <Grid container spacing={4}>
+    <Grid item>{children}</Grid>
+  </Grid>
```

Comment:
Hm! This piqued my interest. In what way does this grid (that has only one item in it) affect the layout? Is the spacing added on the outside of items too, so it effectively acts as `margin` on the item?

Reply:
I tried setting the width of the contents of the card so that it would not stretch to the full page width. That did not work, so I opted for this approach. Any other suggestions? Is is a bug in the InfoCard? It works for the height as you can see.